avidemux/common/ADM_render/GUI_sdlRender.cpp

```diff
@@ -51,7 +51,9 @@ static std::vector<sdlDriverInfo>listOfSDLDrivers;
 class sdlRenderImpl: public VideoRenderBase
 {
   protected:
+              GUI_WindowInfo info;
```

Comment:
I guess we don't need the full struct, just a double for scalingFactor, do we?

Reply:
maybe in the future other members will be needed. there is literally no gain if we only store the scalingFactor